.circleci/config.yml

```diff
@@ -210,26 +210,48 @@ jobs:
       - android/accept-licenses
       - android-wordpress-orb/restore-gradle-cache
       - android/restore-build-cache
-      - run:
-          name: Replace API_KEY
-          working_directory: examples/purchase-tester
-          command: |
-            sed -i s/api_key/$API_KEY/ src/main/java/com/revenuecat/purchasetester/MainApplication.kt
       - run:
           name: Prepare Keystore
           working_directory: examples/purchase-tester
           command: echo $RELEASE_KEYSTORE | base64 -d > keystore
       - run:
           name: Assemble
           command: |
-            ./gradlew -p examples/purchase-tester assemble \
+            ./gradlew -p examples/purchase-tester assembleLatestDependenciesRelease \
             -PreleaseKeyAlias=$RELEASE_KEY_ALIAS \
             -PreleaseKeystorePassword=$RELEASE_KEYSTORE_PASSWORD \
             -PreleaseKeyPassword=$RELEASE_KEY_PASSWORD
       - store_artifacts:
-          path: examples/purchase-tester/build/outputs/apk/release/purchase-tester-release.apk
+          path: examples/purchase-tester/build/outputs/apk/latestDependencies/release/purchase-tester-latestDependencies-release.apk
+      - android-wordpress-orb/save-gradle-cache
+      - android/save-build-cache
+
+  publish-purchase-tester-latest-dependencies-release:
+    executor: android-executor
+    steps:
+      - checkout
+      - install-sdkman
+      - revenuecat/install-gem-unix-dependencies:
+          cache-version: v1
+      - android/accept-licenses
+      - android-wordpress-orb/restore-gradle-cache
+      - android/restore-build-cache
+      - run:
+          name: Prepare Keystore
+          working_directory: examples/purchase-tester
+          command: echo $PURCHASE_TESTER_RELEASE_KEYSTORE | base64 -d > keystore
```

Comment:
We already have a `RELEASE_KEYSTORE` and other envs with different signing information, but that signing info is not the one needed to upload to the play store. I left the other info for the `assemble-sample-app` job, but that build can't be uploaded to the play store so we might want to update it there as well.